crates/bevy_asset/src/server/mod.rs

```diff
@@ -1190,16 +1194,17 @@ pub(crate) enum InternalAssetEvent {
 }
 
 /// The load state of an asset.
-#[derive(Component, Copy, Clone, Debug, Eq, PartialEq, Ord, PartialOrd)]
+#[derive(Error, Component, Clone, Debug, PartialEq, Eq)]
```

Comment:
This type shouldn't impl `Error`: it's analgous to a `Result`. Sorry for any miscommunication.